variant_test.go

```diff
@@ -4,10 +4,25 @@ import (
 	"reflect"
 	"testing"
 	"time"
+	"github.com/shopspring/decimal"
```

Comment:
Sorry for not being clear @sudomake . I meant the space between the two external dependencies, like so:

```
"time"

"github.com/shopspring/decimal"
httpmock "gopkg.in/jarcoal/httpmock.v1"
```
So we have the standard library imports grouped by themselves and external imports grouped by themselves.